tests/containers/privileged_mode.pm

```diff
@@ -37,6 +37,11 @@ sub run {
     # Capabilities are only available in privileged mode
     my $capbnd = script_output("cat /proc/1/status | grep CapBnd");
     validate_script_output("$runtime run --rm --privileged $image cat /proc/1/status | grep CapBnd", sub { m/$capbnd/ });
+
+    # Podman inside the container
+    assert_script_run("$runtime run -it --privileged --name outer-container registry.suse.com/bci/bci-base:latest /bin/bash");
```

Comment:
Let's use a variable for the image name.

Reply:
done :)